plone/outputfilters/browser/resolveuid.py

```diff
@@ -27,11 +28,8 @@ def uuidToURL(uuid):
     "But be aware that this does an extra security check."
 )
 def uuidToObject(uuid):
```

Comment:
Please deprecate this function with a decorator -> https://docs.plone.org/develop/styleguide/deprecation.html#deprecating-methods-and-properties

Reply:
Have a look, are these the changes needed?